lib/measures/AddSimplePvToShadingSurfacesByType/measure.xml

```diff
@@ -155,17 +155,5 @@
       <usage_type>script</usage_type>
       <checksum>7F195550</checksum>
     </file>
-    <file>
-      <filename>PV_test_model/run.db</filename>
-      <filetype>db</filetype>
-      <usage_type>test</usage_type>
-      <checksum>64280EEA</checksum>
-    </file>
-    <file>
-      <filename>PV_test_model/files/USA_CO_Golden-NREL.724666_TMY3.epw</filename>
```

Comment:
not sure if this is okay... seems like maybe the PV_test_model folder was supposed to be there?

Reply:
The directory named `PV_test_model` can be removed. Someone (likely me) probably edited the OSM in os app, which made directory, and then forgot to remove directory before updating measure xml and making commit

Reply:
I just removed them, retested locally, and updated the XML, and made commit.